core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -54,6 +59,30 @@ object Boot extends App with LazyLogging {
       slickDataSource.initWithLiquibase(liquibaseChangeLog, changelogParams)
     }
 
+    val metricsConf = conf.getConfig("metrics")
```

Comment:
I didn't see unit tests for the config parsing stuff but I verified it manually with a rawls.conf file.